script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2.txt

```diff
@@ -25,7 +25,7 @@ For some reason, I imagine a funeral with no attendees or a black and white stri
 I decide to shake off my unpleasant premonition and head to the classroom.
 <Page7>
 I enter the empty classroom.
-I sit down on my chair and kill time until the morning homeroom.
+I sit down in my chair and kill time until the morning homeroom.
```

Comment:
Beta_testers_suggestions_2021_12_05 already changed that once, from "in" to "on".
Is it "sit down on my chair" or "sit down in my chair"?